deployment/cronjob.yaml

```diff
@@ -1,40 +1,70 @@
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
 kind: CronJob
 metadata:
   name: charts-syncer
 spec:
   schedule: "*/30 * * * *"
+  # If a job is scheduled wait for it to finish
+  concurrencyPolicy: Forbid
   jobTemplate:
     spec:
       template:
         spec:
-          restartPolicy: OnFailure
           containers:
-          - name: c3styncer
+          - name: charts-syncer
             imagePullPolicy: Always
-            image: gcr.io/bitnami-labs/charts-syncer:TAG
-            args: ["sync", "--config", "/charts-syncer.yaml", "-v", "4"]
-            # env:
-            #   - name: SOURCE_AUTH_USERNAME
-            #     valueFrom:
-            #       secretKeyRef:
-            #         name: charts-syncer-credentials
-            #         key: source-username
-            #   - name: SOURCE_AUTH_PASSWORD
-            #     valueFrom:
-            #       secretKeyRef:
-            #         name: charts-syncer-credentials
-            #         key: source-password
-            #   - name: TARGET_AUTH_USERNAME
-            #     valueFrom:
-            #       secretKeyRef:
-            #         name: charts-syncer-credentials
-            #         key: target-username
-            #   - name: TARGET_AUTH_PASSWORD
-            #     valueFrom:
-            #       secretKeyRef:
-            #         name: charts-syncer-credentials
-            #         key: target-password
+            image: gcr.io/bitnami-labs/charts-syncer:v0.14.0
```

Comment:
Do we want to hardcode this version here? We will probably forget to update it with new releases

Reply:
I want to make sure it works and reduce the min number of manual steps.

You are right that not having automation to maintain this up to date is not great. I was thinking about using smth like [this](https://github.com/marketplace/actions/create-pull-request) in the release pipeline which will modify the entry and create a PR.

Would that work?

Reply:
Not sure...would that imply that for each release we do there will be a new PR that we have to manually merge? Or it can be merged automatically?

Reply:
> we have to manually merge? Or it can be merged automatically?

Either, whatever we prefer. I like the PR approach since it allow us some oversight on potential "rewrite" issues while preventing us from forgetting about the change.

Reply:
Ok! LGTM!